srcpkgs/synapse/template

```diff
@@ -31,6 +32,12 @@ make_dirs="
 	/etc/synapse 755 synapse synapse
 	/var/log/synapse 0755 synapse synapse"
 
+do_check() {
+	mkdir -p test
+	unzip ${make_install_target} -d test
+	PYTHONPATH=test pytest3 ${make_check_target}
+}
+
```

Comment:
an easier way to make check work would be to add `synapse` to `checkdepends` (wich makes `$depends` no longer necessary)

Reply:
That would create a build cycle which is not acceptable.

Reply:
I see

maybe this then?

```
pre_check() {
	pip install ${make_install_target}
}
```

Reply:
That would pollute the masterdir and wouldn't clean up

Reply:
true... hmm.

my last suggestion:

```
pre_check() {
	pip install -t ./ --no-deps ${make_install_target}
}
```

this produces some warnings because some directories already exist but won't pollute masterdir.

Reply:
For posterity, I just copied this from https://github.com/void-linux/void-packages/blob/b79b3d30270b315301507ff715bb137bdf499d98/srcpkgs/python3-quart/template#L20-L28

I pushed (IMO) a better solution which only extracts the dist-info, which is all that is required.

Reply:
We might want to do something similar in the pep517 build style.